src/scenes/home/thanks/thanks.css

```diff
@@ -0,0 +1,10 @@
+.auth {
+	padding-top: 2%;
```

Comment:
We need to avoid paddings and Marigns with percentages for now as they do not work with flexbox on Firefox.